com.unity.render-pipelines.universal/Runtime/PostProcessPasses.cs

```diff
@@ -30,7 +52,12 @@ internal struct PostProcessPasses : IDisposable
 
         public bool isCreated { get => m_CurrentPostProcessData != null; }
 
-        public PostProcessPasses(PostProcessData rendererPostProcessData, Material blitMaterial)
+        /// <summary>
+        /// Creates post process passes with supplied data anda params.
+        /// </summary>
+        /// <param name="rendererPostProcessData">Post process resources.</param>
+        /// <param name="postProcessParams">Post process run-time creation parameters.</param>
+        public PostProcessPasses(PostProcessData rendererPostProcessData, ref PostProcessParams postProcessParams)
```

Comment:
```suggestion
        public PostProcessPasses(PostProcessData rendererPostProcessData, in PostProcessParams postProcessParams)
```

Imo we should use `in` which is the C# const reference. The reason we have several signatures with `ref` today even though they should be immutable is because `in` was not available in C# by the time we started URP development.